Stratis.Bitcoin/BlockStore/BlockStoreFeature.cs

```diff
@@ -57,13 +58,16 @@ public static class MBlockStoreBuilderExtension
 	{
 		public static IFullNodeBuilder UseBlockStore(this IFullNodeBuilder fullNodeBuilder)
 		{
-			fullNodeBuilder.ConfigureFeature(features =>
+            var blockRepository = new BlockRepository(fullNodeBuilder.Network, new DataFolder(fullNodeBuilder.NodeSettings));
```

Comment:
BlockRepository can to be injected, do the same as DbCoinView take DataFolder in an additional constructor.
DataFolder is already injected to the DI